python/ray/data/_internal/execution/streaming_executor_state.py

```diff
@@ -266,14 +280,8 @@ def summary_str(self, resource_manager: ResourceManager) -> str:
         ):
             desc += " [backpressured]"
 
-        # Active/pending actors
-        active = self.op.num_active_actors()
-        pending = self.op.num_pending_actors()
-        if active or pending:
-            actor_str = f"; Actors: {active}"
-            if pending > 0:
-                actor_str += f", (pending: {pending})"
-            desc += actor_str
+        # Actors info
+        desc += self.actor_info_progress_str()
```

Comment:
oh, sorry, I meant just adding this `actor_info_progress_str` in `PhysicalOperator` and get rid of the `num_xxx_actors` methods. Because it seems a bit overkill to have so many methods and indirections